doc/IO.org

```diff
@@ -80,27 +80,38 @@ will become the name of the object.  Otherwise, objects will be given arbitrary
 names.
 
 
-** FCA containing context, lattice and implication sets
+** Store a complete FCA data study: context, lattice and implication sets
 
-An FCA contains the context and can contain the corresponding lattice and several 
-implication sets as well. The I/O functions are ~read-fca~ and ~write-fca~. The 
+An FCA data format contains the context and can contain the corresponding lattice 
+and several implication sets as well. The I/O functions are ~read-fca~ and ~write-fca~. The 
 standard format is the ~:json~ format. See [[../testing-data/digits-fca.json][digits-fca.json]] for an example 
 of the format.
 
 An FCA can be read using
 
 #+begin_src clojure
-(read-fca "testing-data/digits-fca.json")
+(read-fca "testing-data/small-fca.json")
 #+end_src
 
 or
 
 #+begin_src clojure
-(read-fca "testing-data/digits-fca.json" :json)
+(read-fca "testing-data/small-fca.json" :json)
 #+end_src
 
 The reading function returns a map containing the keys ~:context~, ~:lattice~ 
```

Comment:
And properties like what basis is used? More detail on the format would be nice.

Reply:
More properties for the `:implication-sets` key will be added when Implication Sets are added as an object.